Setup/DeployPrivateKudu.cmd

```diff
@@ -1,48 +1,58 @@
-@if "%_echo%" neq "1" @echo off
```

Comment:
But why is the diff showing every line? It's usually the sign of some line ending issue.